src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -55,74 +68,32 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
     [categories]
   );
 
-  const authorList = authors?.join(", ");
-
   const userCount = posterVenue.recentUserCount ?? 0;
-  const hasUsers = userCount > 0;
-  const userCountText = `${userCount} ${
-    userCount === 1 ? "current visitor" : "current visitors"
-  }`;
-
-  const renderInfoLink = useCallback(
-    (url: string) => (
-      <a href={url} {...externalUrlAdditionalProps}>
-        {url.replace(/(^\w+:|^)\/\//, "")}
-      </a>
-    ),
-    []
-  );
 
-  const renderMoreInfoUrl = useMemo(() => {
-    if (!moreInfoUrl) return;
-
-    return renderInfoLink(moreInfoUrl);
-  }, [moreInfoUrl, renderInfoLink]);
-
-  const renderMoreInfoUrls = useMemo(() => {
-    if (!moreInfoUrls) return;
-
-    return moreInfoUrls.map((infoUrl) => (
-      <div key={infoUrl}>{renderInfoLink(infoUrl)}</div>
-    ));
-  }, [moreInfoUrls, renderInfoLink]);
-
-  const hasMoreInfo = renderMoreInfoUrl || renderMoreInfoUrls;
+  const thumbnailStyles = useCss({
+    backgroundImage: `url(${thumbnailUrl})`,
+  });
```

Comment:
There is a check that can be done for invalid image ( `useValidImage()`) used in few `useCss()` places in the project

Reply:
Resolved in 461139c50